src/peer-store/index.js

```diff
@@ -195,15 +111,16 @@ class PeerStore extends EventEmitter {
   }
 
   /**
-   * Find the stored information of a given peer.
+   * Get the stored information of a given peer.
    * @param {PeerId} peerId
    * @returns {peerInfo}
    */
-  find (peerId) {
+  get (peerId) {
     if (!PeerId.isPeerId(peerId)) {
       throw errcode(new Error('peerId must be an instance of peer-id'), ERR_INVALID_PARAMETERS)
     }
 
+    const id = this.peerIds.get(peerId.toString())
```

Comment:
B58. We need to switch everything together, otherwise we'll likely end up with a bug where we have the same peer stored with the wrong string id.
```suggestion
    const id = this.peerIds.get(peerId.toB58String())
```